packages/discord.js/src/util/Colors.ts

```diff
@@ -1,6 +1,4 @@
-'use strict';
-
-module.exports = {
+export const Colors = {
```

Comment:
Why are many things an enum, but this one isn't?

Reply:
It does align more with how it was before, and I don't see why we'd need a reverse mapping

Reply:
I'm sticking with `const` here because a color can be any `number`, this structure just defines some predetermined ones.